Python/sort/Parallel_MergeSort.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+""" Solution: Sort an array of random integers with merge sort using parallel approach."""
+
+import random
+import math
+import multiprocessing as mp
+
+""" helper method to merge two sorted subarrays
+    array[l..m] and array[m+1..r] into array """
+def merge(array, left, mid, right):
+    # copy data to temp subarrays to be merged
+    left_temp_arr = array[left:mid+1].copy()
+    right_temp_arr = array[mid+1:right+1].copy()
+    # initial indexes for left, right and merged subarrays
+    left_temp_index = 0
+    right_temp_index = 0
+    merge_index = left
+    # merge temp arrays into original
+    while (left_temp_index < (mid - left + 1) or right_temp_index < (right - mid)):
+        if (left_temp_index < (mid - left + 1) and right_temp_index < (right - mid)):
+            if (left_temp_arr[left_temp_index] <= right_temp_arr[right_temp_index]):
+                array[merge_index] = left_temp_arr[left_temp_index]
+                left_temp_index += 1
+            else:
+                array[merge_index] = right_temp_arr[right_temp_index]
+                right_temp_index += 1
+        elif (left_temp_index < (mid - left + 1)): # copy any remaining on left side
+            array[merge_index] = left_temp_arr[left_temp_index]
+            left_temp_index += 1
+        elif (right_temp_index < (right - mid)): # copy any remaining on right side
+            array[merge_index] = right_temp_arr[right_temp_index]
+            right_temp_index += 1
+        merge_index += 1
+
+""" parallel implementation of merge sort """
+def par_mergesort(array, *args):

```

Comment:
update`def par_mergesort(array, *args):` to `def parallel_mergesort(array, *args)`